composer.json

```diff
@@ -109,13 +109,15 @@
     "extra": {
         "installer-name": "Framework",
         "neos": {
-            "warning": "AUTOGENERATED FILE, ONLY MODIFY THE .composer.json IN THIS DIRECTORY AND RUN THE MERGE SCRIPT."
+            "warning": "AUTOGENERATED FILE, ONLY MODIFY THE .composer.json IN THIS DIRECTORY AND RUN THE COMPOSER MANIFEST MERGER SCRIPT PROVIDED BY THE BUILD ESSENTIALS PACKAGE."
         }
     },
     "require-dev": {
         "mikey179/vfsstream": "~1.6",
         "phpunit/phpunit": "~9.0",
-        "vimeo/psalm": "~4.1.1"
+        "vimeo/psalm": "~4.1.1",
+        "doctrine/orm": "^2.6",
```

Comment:
Because https://github.com/neos/flow-development-collection/blob/master/Neos.Utility.ObjectHandling/composer.json#L12-L13 :/

Reply:
It was added by @kitsunet with https://github.com/neos/flow-development-collection/commit/61db144975288bcce132afc976d3da1db8f5ccda#diff-f359238194d5b447aef1e1e4f3ccb42351c95b40fb00524466e40b4ab36a3cb9R12-R13 but I doubt that this should be a **dev** dependency

Reply:
Yeah well... technically it is, because otherwise the tests won't run. Given probably no one uses ObjectHandling out of a Flow context we're safe with that. For the other potential uses it would be better to make those tests optional.